suites/tipset/message_dedup.go

```diff
@@ -24,71 +24,81 @@ func TestBlockMessageDeduplication(t *testing.T, factory state.Factories) {
 	t.Run("apply a single BLS message", func(t *testing.T) {
 		td := builder.Build(t)
 		defer td.Complete()
-		blkBuilder := drivers.NewTipSetMessageBuilder(td)
+		tipB := drivers.NewTipSetMessageBuilder(td)
+		blkB := drivers.NewBlockBuilder(td.ExeCtx.Miner)
 
 		sender, _ := td.NewAccountActor(address.SECP256K1, big_spec.NewInt(10_000_000))
 		receiver, _ := td.NewAccountActor(address.SECP256K1, big_spec.Zero())
 
-		blkBuilder.WithTicketCount(1).
-			// send value from sender to receiver
-			WithBLSMessageOk(
-				td.MessageProducer.Transfer(receiver, sender, chain.Nonce(0), chain.Value(big_spec.NewInt(100))),
-			).ApplyAndValidate()
+		tipB.WithBlockBuilder(
+			blkB.WithTicketCount(1).
```

Comment:
In a follow-up, please set a default of 1 or make it a ctor param, so we can drop this line.